### PR TITLE
Ironmonk88 chat message changes

### PR DESCRIPTION
**Pull request by Dosendusche**
- resolved merge conflicts.

I think it would be good if someone could take a look if that would still work like this especially ironmonk88. The rolling in starfinder alone still works fine.

### CODE REVIEW

**Comment by LebombJames** [comment]
How come this PR doesn't have the if statement around the chat message creation in dice.js?
image
image

**Comment by LebombJames** [comment]
Wait nvm checking the full file I can see what's going on

File: src/module/dice.js [hunk 1]
```diff
@@ -262,9 +262,9 @@ export class DiceSFRPG {
             const itemContext = rollContext.allContexts['item'];
             const htmlData = [{ name: "rollNotes", value: itemContext?.system?.rollNotes }];
 
-            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards");
+            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards") && chatMessage;
```
Comment (LebombJames) [inline src/module/dice.js:265]: This check is needless given lines 267, 288, 757 and 783 check for `chatMessage` anyway

**Review by LebombJames — CHANGES_REQUESTED** [review]

File: src/module/dice.js (hunk 1 above)
Comment (Dosendusche) [reply src/module/dice.js]: done. thanks for the hint.

File: src/module/dice.js [hunk 2]
```diff
@@ -752,10 +752,10 @@ export class DiceSFRPG {
             htmlData.push({ name: "damage-parts", value: JSON.stringify(tempParts) });
             htmlData.push({ name: "rollNotes", value: itemContext?.data?.damageNotes });
 
-            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards");
+            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards") && chatMessage;
```
Comment (LebombJames) [inline src/module/dice.js:755]: Didn't spot this one too, my bad

**Review by LebombJames — CHANGES_REQUESTED** [review]

File: src/module/dice.js (hunk 2 above)
Comment (Dosendusche) [reply src/module/dice.js]: yep, same. I could have done it before.

**Pull request merged**